tests/integration/test_device_mgmnt_interface.py

```diff
@@ -0,0 +1,53 @@
+'''Wakaama integration tests (pytest)'''
+import re
+import json
+
+
+def test_read_on_object(lwm2mserver, lwm2mclient):
+    """LightweightM2M-1.1-int-222
+    Purpose of this test is to show conformance with the LwM2M Read
+    operation on whole Object"""
+
+    lwm2mclient.waitfortext("STATE_READY")
+    # Test Procedure 1
+    assert lwm2mserver.commandresponse("read 0 /1", "OK")
+    text = lwm2mserver.waitforpacket()
+    # Pass-Criteria A
+    assert text.find("COAP_205_CONTENT") > 0
+    packet = re.findall(r"(\[.*\])", text)
+    parsed = json.loads(packet[1])
+    assert next(item for item in parsed if item["n"] == "0/0")["v"] == 123
+    assert next(item for item in parsed if item["n"] == "0/0")["bn"] == "/1/"
+    assert next(item for item in parsed if item["n"] == "0/1")["v"] == 300
+    assert next(item for item in parsed if item["n"] == "0/2")["v"] == 0
+    assert next(item for item in parsed if item["n"] == "0/3")["v"] == 0
+    assert next(item for item in parsed if item["n"] == "0/5")["v"] == 0
+    assert next(item for item in parsed if item["n"] == "0/6")["vb"] is False
+    assert next(item for item in parsed if item["n"] == "0/7")["vs"] == 'U'
```

Comment:
This looks like very repetitive code. It's hard to get what should happen here. Some helper function may make the code cleaner to read.

Reply:
ok!